shared.sh

```diff
@@ -2,6 +2,9 @@
 # shellcheck disable=SC2034
 set -euo pipefail
 
+# Force UTF-8 for all commands, for Git-Bash on Windows compatibility
+export LC_ALL=C.UTF-8
```

Comment:
`perl` on MacOS doesn't like this - it emits some warnings ... but I think it's okay if all the tests pass anyway!  😈